package.json

```diff
@@ -44,7 +44,7 @@
     "electron-window-state": "4.1.1",
     "history": "4.7.2",
     "jitsi-meet-electron-utils": "github:jitsi/jitsi-meet-electron-utils#1972c3bf0884ace68eb496894dabae593d6dbf49",
-    "js-md5": "0.7.3",
+    "js-utils": "github:akshitkrnagpal/js-utils#avatar",
```

Comment:
I'll update this when PR in js-utils is merged.

Reply:
:+1:

Reply:
@AkshitKrNagpal the js-utils PRs is merged!